tests/test_model.py

```diff
@@ -97,10 +97,23 @@ def test_predict(preprocess_data):
 
 
 def test_load_and_save():
-    model = modelgym.Model()
+    model = modelgym.Model(learning_task=TASK_CLASSIFICATION)  # model to save and then read
     filepath = "data/newtree.yaml"
```

Comment:
would be really bad if user had this file (data/newtree.yaml) before running the test... instead use tempfile.mkstemp(suffix=".test.yaml") or TemporaryFile
https://security.openstack.org/guidelines/dg_using-temporary-files-securely.html